source/elements/oneDAL/include/oneapi/dal/memory.hpp

```diff
@@ -0,0 +1,27 @@
+namespace oneapi::dal {
+
+/// @tparam Data The type of elements to be freed
+template <typename Data>
+class empty_delete {
```

Comment:
naming convention - is it okay to use this style (delete is verb) and not say empty_deleter?

Reply:
It's aligned with [std::default_delete](https://en.cppreference.com/w/cpp/memory/default_delete). Boost, however, is using **_deleter** suffix for such types